src/components/screens/consul/Proposals/ProposalDetail.js

```diff
@@ -246,15 +246,13 @@ export const ProposalDetail = ({ data, onRefresh, route, navigation }) => {
 };
 /* eslint-enable complexity */
 
+const styles = StyleSheet.create({
+  imageContainerStyle: { alignSelf: 'center' }
```

Comment:
i think style sheets are never one lined across the whole code base. could you please adjust it here as well?